### ISSUE TITLE
[Question] Stream in a Box ?

### ISSUE THREAD

**Issue by KaKi87** [kenan238/reblessed#7]
Hello,

A `Screen` is initialized with input/output `Stream` objects which are `process.stdin`/`process.stdout` by default.

Would it be possible, inside that `Screen`, to initialize a `Box`, which itself contains a `Stream` ?

The purpose would be to have an SSH connection in a `Box`.

Thanks

**Comment by kenan238** [comment]
Sorry for the late response, life has been going all places, I will look into this right now

**Comment by kenan238** [comment]
Looks like I misunderstood your question; Making a box render to a stream could be quite difficult since reblessed calls the parent screen to render it, and the rendering is done by a program object, I'd rather not make too big changes.

**Comment by KaKi87** [comment]
> Making a box render to a stream

No, I want the opposite : make a stream render in a box.

**Comment by kenan238** [comment]
So you pass in a stream and the box renders it?

**Comment by KaKi87** [comment]
Yes.

**Comment by kenan238** [comment]
Alright, I'll get to it

**Comment by kenan238** [comment]
Hello @KaKi87 would you mind checking the `StreamBox`?

**Comment by KaKi87** [comment]
How do I access it ?

I tried the following, none work :

```js
const streamBox = blessed.streamBox();
```

```js
const streamBox = blessed.StreamBox();
```

```js
const streamBox = blessed.streambox();
```

Thanks

**Comment by kenan238** [comment]
It's not in an npm release

**Comment by kenan238** [comment]
gonna upload it one sec

**Comment by KaKi87** [comment]
No need, I used `git+https://github.com/kenan238/reblessed.git`, but the property still isn't defined. Unless it's in a branch ?

**Comment by kenan238** [comment]
Should work, it works for me on the tests

**Comment by KaKi87** [comment]
Could you share a code snippet ?

**Comment by kenan238** [comment]
It's known the test directory

**Comment by kenan238** [comment]
Inactive

**Comment by KaKi87** [comment]
Hello,

You spent much time creating this feature, but I'm sorry I wasted it, and I feel very bad.

Not only was I unable to use it, but I also found a better approach to my problem in the meantime.

I'll make sure this never happens again, to you nor anyone else from the open source community.

Additionally, I would like to make a donation to you if that's okay and doable, not to make myself feel better or anything, but just out of gratefulness for maintaining this project when no one else does.
